code/modules/mob/living/carbon/human/human_organs.dm

```diff
@@ -26,6 +26,22 @@
 // Takes care of organ related updates, such as broken and missing limbs
 /mob/living/carbon/human/proc/handle_organs()
 
+	// Check for the presence (or lack of) vital organs like the brain.
```

Comment:
I wonder if adding a timed check like that for telling if a mob is unrevivable is a good idea? 
Because it used to be that organs would decay on their own after the containing mob is dead for some time. 

Also, I think it would be important to preserve the existing behaviour and have the mob call death when it loses a vital organ and is clinically dead. Because a lot of things rely on that behaviour. 
For example: 

code/modules/mob/living/carbon/life.dm
```dm
	if(stat != DEAD && !InStasis())
		//Breathing, if applicable
		handle_breathing()

		//Mutations and radiation
		handle_mutations_and_radiation()

		//Chemicals in the body
		handle_chemicals_in_body()

		//Random events (vomiting etc)
		handle_random_events()

		// eye, ear, brain damages
		handle_disabilities()

		handle_immunity()
```
All this stuff will keep ticking during the time you're keeping the mob alive without its vital organs, which may defeat the purpose anyways by causing further damage to existing organs faster than someone can fix them against the revive timer you've set here?

Maybe it would be simpler to just let the mob go into dead state, and rely on organ decay time for telling how long can a corpse be revived?

Reply:
> > Because it used to be that organs would decay on their own after the containing mob is dead for some time.
> 
> I don't think this has ever been the case? Organs don't process while inside an owner, they are processed by Life().
> 

Yeah, you're right about that one. It was probably a change on old ps13. I looked at it again, and it doesn't seem to handle that.

> Re: the unrevivable timer, we have no concept of corpse revival in code currently. That is the point of adding the delay to the point of death. Having the body being dead representing the hard final cutoff point makes more sense to me conceptually in a medical sim that includes the heart stopping than letting people arbitrarily remove `stat = DEAD`.
> 
> > Because a lot of things rely on that behaviour. For example:
> 
> I don't think it's an issue that any of the above continues processing while the mob is missing a vital organ. Processing means the body can be medicated, bleeding is still an issue, etc. Whatever specifically should be halted by missing a brain should be checking for the missing brain. If you have specific concerns from the list, I'm happy to add the checks in this PR.

Well, I'm not going to get into another big debate on that. Might as well let you try it out.